src/ralph/sim_cards/admin.py

```diff
@@ -42,6 +43,7 @@ class SIMCardAdmin(RalphAdmin):
             )
         }),
     )
+    search_fields = ('card_number', 'phone_number', 'pin1', 'puk1')
```

Comment:
Perhaps, it would be reasonable to also display carriers.

Reply:
I added carriers